### DISCUSSION TITLE
How to create create view in django

With class base view and def

### DISCUSSION THREAD

**Discussion by amirhoseinshoajei** [SDWebImage/SDWebImage#3700] [General]
Help me

**Comment by SilentSpectre-arch** [comment]
from django.views.generics import createview
example:

class BlogCreate(LoginRequiredMixin,CreateView):
    model = Blog
    fields = ['title','description']
    context_object_name = 'objects-create'
    template_name = 'blog/blog-create.html'
    success_url = reverse_lazy ('blog-list')

    def form_valid(self,form):
        form.instance.author = self.request.user
        return super().form_valid(form)

**Comment by SilentSpectre-arch** [comment]
answer 2
